book/_toc.yml

```diff
@@ -23,5 +23,6 @@
 - file: tutorials/index
   sections:
     - file: tutorials/raster/intro-ipyleaflet
+    - file: tutorials/core-datasets/core-datasets-tutorial
```

Comment:
thanks @meganmason. looks like you might have forgotten to add core-datasets folder (so just `git add core-datasets`, then `git commit -m 'added core-datasets content`, then git push)